docs/src/main/sphinx/release/release-449.md

```diff
@@ -24,8 +23,9 @@
 * Add support for [the `$partitions` system table](delta-lake-partitions-table). ({issue}`18590`)
 * Add support for reading from and writing to tables with
   [VACUUM Protocol Check](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#vacuum-protocol-check). ({issue}`21398`)
-* Add support for configuring the query retry policy on the S3 filesystem with
-  the `s3.retry-mode` and `s3.max-error-retries` configuration properties.
+* Add support for configuring the query retry policy on the native S3 filesystem
+  with the `s3.retry-mode` and `s3.max-error-retries` configuration properties.
+  ({issue}`21900`)
 * Automatically use `varchar` in struct types as a type during table creation
   when `char` is specified. ({issue}`21511`)
 * Improve performance of writing to Parquet files. ({issue}`22089`)
```

Comment:
remove "Parquet". Delta doesn't have varying file formats, it's only Parquet.

Reply:
Changed to data files as @martint suggested